packages/@okta/vuepress-theme-prose/util/frameworks.js

```diff
@@ -9,6 +9,7 @@ const FRAMEWORK_TO_COMMON_NAME = {
 };
 
 const COMMON_NAME_TO_FANCY_NAME = {
+  apple: 'Apple',
```

Comment:
Why does `SAML` and `Generic OIDC` have a logo in the navigation bar while other IdPs don't?

Reply:
> Why does `SAML` and `Generic OIDC` have a logo in the navigation bar while other IdPs don't?

@alexdahl-okta What do you think?

Reply:
@ZoeMonosson-okta FYI

Reply:
The original design intent was for all IdPs to have an icon in the Stack Selector. We just haven't implemented all of them – no particular reason, just hasn't been prioritized.

Made a new issue to track: #1246.

While it'd be ideal for all of them to implemented, I think my philosophy is progressive enhancement. When we can show an icon, there's value to do so. That value is greater than the value of consistency here (which would mean no icons for any providers).

I don't think it's a blocker, but for Apple and future providers, it'd be great to add an icon as we add the provider.